connector/pom.xml

```diff
@@ -224,6 +224,11 @@
       <artifactId>azure-keyvault-webkey</artifactId>
       <version>1.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.scalatest</groupId>
+      <artifactId>scalatest-funspec_2.11</artifactId>
```

Comment:
See line 191 - duplicate?

Reply:
It is for the funspec I used in my tests, I converted mine to flatSpec and removed that one